src/__test__/backend/dataaccess/McodeV09SmartOnFhirDataSource.test.js

```diff
@@ -83,7 +88,9 @@ describe('SMART on FHIR data source', function() {
           .get('/fhir/Patient?_id=1078857')
           .reply(200, samplePatientSearchData)
           .get('/fhir/Observation?patient=1078857')
-          .reply(200, sampleObservationSearchData);
+          .reply(200, sampleObservationSearchData)
+          .get('/fhir/Encounter/6a8bc97e-3ba1-4fb5-a478-4b5bd888c793') // reference retrieval
+          .reply(200, sampleReferenceResult);
           // in this case it doesn't need to fetch the metadata since the resourceTypes are manually specified
 
         const dataSource = new McodeV09SmartOnFhirDataSource({ resourceTypes: ['Patient', 'Observation'] });
```

Comment:
[DP3] Does this need "Encounter" to be added?

Reply:
"Encounter" does not _need_ to be added here because the Encounter is being fetched due to the fact that it is a reference on one of the entries that does match those types. When we fetch references, we do not check against the specified types (if there are any). I will add a Jira to explore replacing these specified types on props in config.js with just a set list of supported types that is used everywhere.

Reply:
👍